docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,42 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
-## Getting started
+## Quickstart
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <Button variant="solid">Hello World</Button>
+    </CssVarsProvider>
+  );
 }
-
-export default App;
 ```
 
-### Render your first component
+You can play around with this code in the interactive Code Sandbox demo below.
+Try changing the `variant` on the Button to `soft` to see how the style changes:
 
-To render any Joy UI component, make sure you place them inside the `CssVarProvider` element.
+{{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
```

Comment:
From what I understand, we also need to load the default font for Joy UI.

Reply:
Don't we also miss it from the codesandbox export of the demos? e.g. https://codesandbox.io/s/c50w6x?file=/public/index.html. I don't personally have Public Sans.

Reply:
Following up, my understanding is that Public Sans is bundled directly into `@mui/joy` so there's no need to load it in. Is that correct @siriwatknp?

Reply:
@samuelsycamore fonts can't be bundled directly, they have to be installed somehow. For example

- default Joy UI: https://codesandbox.io/s/misty-sky-n5ct34?file=/public/index.html
- with Public Sans installed: https://codesandbox.io/s/cocky-herschel-3gk5k5?file=/public/index.html

To compare with https://mui.com/joy-ui/react-typography/#levels. I feel like we might not actually want to use Public Sans, or June and Danilo have the font already installed on their OS so they were blind to the problem.

Reply:
@oliviertassinari that makes sense. I'd like to get this PR merged, and I think we should continue the conversation about the font somewhere else. How should we proceed to close this out? We could:

1. leave the demo as is and don't mention the font in the doc (for now, assuming it's going to change very soon)
2. import Public Sans in the demo via Google Fonts, and add info about installing the font to Joy UI's Installation page as we've done for Roboto with Material UI

The first is the fastest and simplest option, but I'd go with the second option in the interest of making these docs fully functional now, whether or not Public Sans or its installation change in the future. We'll also want to update the [Joy UI templates](https://mui.com/joy-ui/getting-started/templates/) to replace [the MD/Roboto stuff](https://codesandbox.io/s/rjnzeb?file=/public/index.html) with Public Sans.

cc: @siriwatknp @danilo-leal

Reply:
I'd be down to go with the first option there first, taking advantage of Joy's still in-alpha state to iterate on the typeface but would definitely understand if you understand is better to document it already and then, later on, we would just switch over to the new/different typeface if that happens, and whatever it ends up being!

Reply:
@danilo-leal If we go with option 1, should we remove Public Sans from the source? This way, we will have a more predictable font experience. I mean, this way, the font displayed won't change based on whether I have "Public Sans" installed on my OS or not.

Reply:
Personally, I think I would be kind of frustrated if I was trying out Joy UI for a project and couldn't find any info about fonts (aside from what's on the Typography page)—especially if I'm coming from Material UI, where that question is clearly answered in the docs. Even if it's still in alpha and could potentially change, I'd rather have a solution that works now.

Reply:
Alright, that definitely makes sense. Let's add documentation around the font now, keep it in the source, and then if that changes at any time, we just update everything to the new typeface. Does that sound good?